### ISSUE TITLE
Webdriver times out no matter the defined timeout for page load

### ISSUE THREAD

**Issue by Rantzur1992** [testproject-io/csharp-opensdk#100]
The Tests and imports
```xml
-------Reference -----------

<ItemGroup>
    <PackageReference Include="DotNetSeleniumExtras.WaitHelpers" Version="3.11.0" />
    <PackageReference Include="NUnit" Version="3.12.0" />
    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
    <PackageReference Include="System.Data.OleDb" Version="5.0.0" />
    <PackageReference Include="TestProject.OpenSDK" Version="0.65.1" />
    <PackageReference Include="TestProject.OpenSDK.SpecFlowPlugin" Version="0.65.1" />
  </ItemGroup>
    
  -------------Exception details --------------------------
    Message: 
    OpenQA.Selenium.WebDriverException : The HTTP request to the remote WebDriver server for URL http://127.0.0.1:52519/wd/hub/session/6742ede29610cbfd11c3208976315fa6/element timed out after 10 seconds.
      ----> System.Net.WebException : The operation has timed out.
  Stack Trace: 
    HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)
    HttpCommandExecutor.Execute(Command commandToExecute)
    CustomHttpCommandExecutor.Execute(Command commandToExecute, Boolean skipReporting)
    CustomHttpCommandExecutor.Execute(Command commandToExecute)
    BaseDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
    RemoteWebDriver.FindElement(String mechanism, String value)
    RemoteWebDriver.FindElementByXPath(String xpath)
    <>c__DisplayClass19_0.<XPath>b__0(ISearchContext context)
    By.FindElement(ISearchContext context)
    RemoteWebDriver.FindElement(By by)
    DriverManager.Test1(String uname, String pwd, String loggedinUserData) line 45
    --WebException
    HttpWebRequest.GetResponse()
    HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)
	
-------------------------------------------------------------------------------
```

```csharp
public void Test1(string uname, string pwd, string loggedinUserData)
{
    //OpenQA.Selenium.Chrome.ChromeOptions options = new OpenQA.Selenium.Chrome.ChromeOptions();
    //options.AddArgument("no-sandbox");
    
    driver = new ChromeDriver(projectName:"OpenSDK",jobName:"Trail");           
    driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromMinutes(5);
    
    try
    {
        driver.Navigate().GoToUrl("URL HERE");
    }
    catch(Exception e)
    {
        Console.WriteLine(e.Message);
        driver.Quit();
    }
    
    var wait = new WebDriverWait(driver, new TimeSpan(0, 2, 0));
    IWebElement element = wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementIsVisible(By.XPath("//input[@data-qa-id='login-email-input']")));
    element.SendKeys(uname);
    
    driver.FindElement(By.XPath("//button[@data-qa-id='next-button']")).Click();
    driver.FindElement(By.XPath("//input[@data-qa-id='login-password-input']")).SendKeys(pwd);
    driver.FindElement(By.XPath("//button[@data-qa-id='login-button']")).Click();
    
    element = wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementIsVisible(By.XPath("//button[.=' Geetha Vajrala']")));
    string loggedInUser = element.Text;
    Assert.AreEqual(loggedInUser, loggedinUserData);
    
    driver.FindElement(By.XPath("//a[normalize-space()='Classes & Properties']")).Click();
    Thread.Sleep(5000);
}
```

Note, this does not happen while using the webdriver from selenium.

**Comment by basdijkstra** [comment]
I'll see what I can do to reproduce and fix this. Is the actual URL relevant to this situation, @Rantzur1992 ? Because you removed it from the code. Or will this happen with any website?

**Comment by Rantzur1992** [comment]
This can happen on every website which load time is long

**Comment by Rantzur1992** [comment]
@basdijkstra Any findings?

**Comment by basdijkstra** [comment]
Not yet. I've been busy fixing other issues and implementing the mobile support. I'll keep you updated.

**Comment by basdijkstra** [comment]
@Rantzur1992 I need a sample website here so I can reproduce this issue. Without that it is impossible for me to pinpoint what is causing the issue.

Using both implicit and explicit waits in the same test probably isn't helping here, either.. But that's a different discussion.

**Comment by Rantzur1992** [comment]
@basdijkstra This happened on a user's Website, I can try to get the URL from them.

**Comment by basdijkstra** [comment]
Thanks, that would be great.

**Comment by basdijkstra** [comment]
Any response yet, @Rantzur1992 ?

**Comment by SantoshYakkala** [comment]
Dear @basdijkstra, 

The issue reported website is not release product yet into the market. Afraid to share the details as it breaches the confidentiality. Issue can be repro on the metered internet or kind of very slow loading of webpage.

**Comment by basdijkstra** [comment]
I'll see if I can find an example.

**Comment by Rantzur1992** [comment]
Hello @SantoshYakkala 
I was not able to find a website that reproduces it without using your own.
Perhaps you can share with me the web application (I know it is not in production yet) but it will save us a lot of time searching so we will solve the issue quicker.

**Comment by Rantzur1992** [comment]
I am closing this, as it requires the URL on the application.

**Comment by s-glatshtein** [comment]
This issue was resolved in the release of 1.2.1, and can now be closed

**Comment by SantoshYakkala** [comment]
> This issue was resolved in the release of 1.2.1, and can now be closed

@s-glatshtein Yes, issue was resolved in v1.2.1 and tested. Functionality is working as expected.
